keras_cv/layers/preprocessing/base_image_augmentation_layer_test.py

```diff
@@ -148,15 +151,22 @@ def test_augment_image_and_localization_data(self):
         images = np.random.random(size=(8, 8, 3)).astype("float32")
         bounding_boxes = np.random.random(size=(3, 5)).astype("float32")
         keypoints = np.random.random(size=(3, 5, 2)).astype("float32")
+        segmentation_mask = np.random.random(size=(8, 8, 3)).astype("float32")
```

Comment:
The shape of the mask seems wrong. The last dim should be 1 (the class). If you get the mask from the raw data, then it should be processed first to convert the RGB value to the class index.

Reply:
The last dim is arbitrary, since it can be any number representing e.g `classes`.

But I think it was confusing having it as 3 because it somewhat implies that it represents color channels, so I've updated it to 2.